src/index.html

```diff
@@ -9,7 +9,141 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
   <script type="text/javascript" src="./main.js"></script>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST" autocomplete="off" onsubmit="onSubmit()">
+    <fieldset class="fiel">
+      <legend>Personal information</legend>
+      <p>
+        <label for="surname">Surname:</label>
+        <input
+        type="text"
+        id="surname"
+        name="surname">
```

Comment:
```suggestion
        <input
          type="text"
          id="surname"
          name="surname"
        >
```
attributes must have the same indentations as nested tags